src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ImportStatusBauServiceTests.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Model;
+using Microsoft.Azure.Cosmos.Table;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.TableStorageTableNames;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class ImportStatusBauServiceTests
+    {
+        private static ImportStatusBauService BuildImportStatusBauService(List<DatafileImport> queryResults)
+        {
+            var tableStorageService = new Mock<ITableStorageService>(MockBehavior.Strict);
+            tableStorageService.Setup(storageService =>
```

Comment:
Not a good idea to set up mock expectations in the build/setup method unless those expectations are common to all service methods (not likely in most cases).

Better to pass in the mock as a parameter and set its expectation up ad-hoc in each test case.